docs/pages/overview.md

```diff
@@ -20,13 +20,15 @@ Meshery provides this high-level functionality:
 
 
 <h2>What challenges does Meshery solve?</h2>
-<p style="text-align:center;"><b>Service mesh management - one or multiple service meshes.</b></p>
```

Comment:
This was intentionally centered so as to provide the reader with a visual break from the rest of the content, but we can make this change.

Reply:
Center aligned looks a bit off, this looks much better like a one-liner of what challenges meshery solve?
<img width="815" alt="Screenshot 2020-05-20 at 3 27 42 PM" src="https://user-images.githubusercontent.com/44091822/82433063-85eb5180-9aae-11ea-9bbf-938e8eddb288.png">

Reply:
SGTM